docs/LLVMIntrinsicFunctionLowering.md

```diff
@@ -67,7 +69,9 @@ in one of four ways:
             return BM->addUnaryInst(OpBitReverse, Ty, Op, BB);
           }
 
-2. Some intrinsics are emulated by basic operations in SPIRVWriter.cpp.  For example:
+Method 2:
+
+   Some intrinsics are emulated by basic operations in SPIRVWriter.cpp.  For example:
```

Comment:
Nit: I think it is better to not refer to cpp files. Here you can just refer to SPIRVWriter pass. Also, please try to avoid using cpp filenames in the text below as well.

Thanks

Reply:
Done.